lib/gpu/lib/src/formats.dart

```diff
@@ -59,3 +59,49 @@ enum TextureCoordinateSystem {
   /// (0, 0) is the top-left of the image with +Y going down.
   renderToTexture,
 }
+
+enum BlendFactor {
+  zero,
+  one,
+  sourceColor,
+  oneMinusSourceColor,
+  sourceAlpha,
+  oneMinusSourceAlpha,
+  destinationColor,
+  oneMinusDestinationColor,
+  destinationAlpha,
+  oneMinusDestinationAlpha,
+  sourceAlphaSaturated,
+  blendColor,
+  oneMinusBlendColor,
+  blendAlpha,
+  oneMinusBlendAlpha,
+}
+
+enum BlendOperation {
+  add,
+  subtract,
+  reverseSubtract,
+}
+
+enum LoadAction {
+  dontCare,
+  load,
+  clear,
+}
+
+enum StoreAction {
+  dontCare,
+  store,
+  multisampleResolve,
+  storeAndMultisampleResolve,
+}
+
+enum ShaderStage {
+  unknown,
+  vertex,
+  fragment,
+  tessellationControl,
+  tessellationEvaluation,
+  compute,
+}
```

Comment:
Can we avoid exposing stages we don't have a plan for yet? I think this could just be vertex and fragment.

Reply:
Done.